CONTRIBUTING.md

```diff
@@ -38,15 +40,15 @@ Follow these instructions for contributing new recipes. The Goal headers indicat
         1. Change directory into your recipe folder: `cd recipes/MyRecipeName`
         1. Install dependencies as normal: `spago install <packageName>`
         1. Return to the root directory: `cd ../..`
-    - **Note**: you can only install dependencies that exist in the latest package set release; you cannot add or override packages in `packages.dhall` (see Principles section for more contxt).
+    - **Note**: you can only install dependencies that exist in the latest package set release; you cannot add or override packages in `packages.dhall` (see Principles section for more context).
 1. Install needed `npm` dependencies via `npm i <packageName>`. These will be installed to the root folder's `node_modules` folder, not a corresponding folder in the recipe.
     - If you do install `npm` dependencies for your recipe, please state which libraries were installed in the recipe's `README.md` file.
 1. Implement your recipe. If you add any new modules, always start the module name with your recipe's "Unique Recipe Name" (e.g. `MyNewRecipe.Foo`, `MyNewRecipe.Module.Path.To.Cool.Types`)
-    - Run `spago -x recipes/MyNewRecipe/spago.dhall build -w` while in the root folder for faster iteration while developing
+    - Run `make MyNewRecipe-build-watch` while in the root folder if you'd like to automatically rebuild recipes upon changes.
```

Comment:
Good tip. I haven't had much luck with purs-ide rebuilds for the cookbook project structure.

Reply:
Might also be good to note `-web` for development with automatic refresh.
Is there a watched `-run` option?

Reply:
If it's using `spago ... -w` underneath, then `run` should work, too. I've also noticed that `parcel` will pick up changes if I rebuild the code.